test/Indice.Services.Tests/BlobPropertiesExtensionsTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Threading.Tasks;
+using Azure;
+using Indice.Extensions;
+using Xunit;
+
+namespace Indice.Services.Tests
+{
+    public class BlobPropertiesExtensionsTest
+    {
+        [Fact]
+        public void EtagWithQuotes() {
```

Comment:
change this to one function with the `[Theory]` attribute and an input parameter.
```c#
        [Theory]
        [InlineData("0x8D979DAB44F6E71")]
        [InlineData("\"0x8D979DAB44F6E71\"")]
        public void EtagWithoutQuotes(string input) {
            ETag sourceEtag = new ETag();
            var etag = sourceEtag.TryWrapEtagWithQuotes(input);
            Assert.NotNull(etag);
            Assert.StartsWith("\"", etag);
            Assert.EndsWith("\"", etag);
        }
```